tests/unit/test_config.py

```diff
@@ -3,72 +3,198 @@
 
 import pytest
 
-from linker.configuration import DEFAULT_ENVIRONMENT, Config
-from linker.step import Step
-from tests.unit.conftest import (
-    ENV_CONFIG_DICT,
-    PIPELINE_CONFIG_DICT,
-    check_expected_validation_exit,
+from linker.configuration import (
+    DEFAULT_ENVIRONMENT,
+    ENVIRONMENT_ERRORS_KEY,
+    INPUT_DATA_ERRORS_KEY,
+    PIPELINE_ERRORS_KEY,
+    Config,
 )
-
-
-def test_config_instantiation(test_dir, default_config):
```

Comment:
This just didn't seem useful anymore. Obviously it instantiates based on all of these other tests and we're doing explicit default attribute checking